src/IncrementalChecker.ts

```diff
@@ -303,7 +282,7 @@ export class IncrementalChecker {
       cancellationToken.throwIfCancellationRequested();
 
       try {
-        this.linter.lint(fileName, undefined, this.linterConfig);
+        this.linter!.lint(fileName, undefined!, this.linterConfig); // REVIEW: this shouldn't work?
```

Comment:
what do you mean by "this shouldn't work?"?

Reply:
`undefined!` 😛